src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      class="search-bar search-bar--big"
```

Comment:
u don't need `--big` modifier, big - its a default state of search bar, so move all styles from big to `search-bar`